crates/modelardb_server/src/parser.rs

```diff
@@ -307,6 +296,7 @@ pub fn tokenize_and_parse_sql(sql: &str) -> Result<Statement, ParserError> {
 /// A top-level statement (SELECT, INSERT, CREATE, UPDATE, etc.) that have been
 /// tokenized, parsed, and for which semantics checks have verified that it is
 /// compatible with ModelarDB. CREATE TABLE and CREATE MODEL TABLE is supported.
+#[derive(Debug)]
```

Comment:
Unclear why `Debug` is added to `ValidStatement`.

Reply:
I cannot use `unwrap_err()` without it.